### ISSUE TITLE
Package request: teehee

### ISSUE THREAD

**Issue by kefiiris** [void-linux/void-packages#50473]
### Package name

teehee

### Package homepage

https://github.com/Gskartwii/teehee

### Description

A modal terminal hex editor inspired by Vim, Kakoune and Hiew

### Does the requested package meet the package requirements?

Compiled

### Is the requested package released?

Yes

**Comment by RotaryBoot58** [comment]
I am going to take a look at it. hopefully if everything goes as expected it should be in the repos soon

**Comment by RotaryBoot58** [comment]
I'm back, late but back. this package seems to have been abandoned maybe? If that is the case I will not maintain it and somebody else would have to do it.

Can you install it with cargo as mentioned in their homepage?

**Comment by classabbyamp** [comment]
> Can you install it with cargo as mentioned in their homepage?

asking questions like this is not useful. *of course* the requester could install any requested package manually, but the point of the package request is to indicate there is interest in seeing it available from xbps.

**Comment by RotaryBoot58** [comment]
> > Can you install it with cargo as mentioned in their homepage?
> 
> asking questions like this is not useful. _of course_ the requester could install any requested package manually, but the point of the package request is to indicate there is interest in seeing it available from xbps.

Yeah you are right, I am so used to help people that do not even check the installation part that now I am saying these stupid things.

But anyway. should a outdated package be added to the repo? it does not have any commit from the last 2 years.
